src/safeds/ml/classical/classification/_classifier.py

```diff
@@ -230,3 +233,42 @@ def f1_score(self, validation_or_test_set: TaggedTable, positive_class: Any) ->
         if (2 * n_true_positives + n_false_positives + n_false_negatives) == 0:
             return 1.0
         return 2 * n_true_positives / (2 * n_true_positives + n_false_positives + n_false_negatives)
+
+    def roc_curve(self, validation_or_test_set: TaggedTable) -> Image:
```

Comment:
Like for `precision` and `recall`, we will need to specify the "positive" class.

Reply:
i don't understand how the "positive" class should be used for the roc_curve